### PR TITLE
Bulk read

### PR DESCRIPTION
**Pull request by cheeetoo**

### CODE REVIEW

File: TeamCode/src/main/java/org/firstinspires/ftc/teamcode/Robot/RobotContainer.java [hunk 1]
```diff
@@ -31,7 +35,7 @@ public RobotContainer(HardwareMap hardwareMap, OpModeType type, Gamepad driveron
         register(claw_subsystem);
         commandScheduler = CommandScheduler.getInstance();
 
-
+        bulkReading = new BulkReading(hardwareMap);
```
Comment (foehammer127) [inline TeamCode/src/main/java/org/firstinspires/ftc/teamcode/Robot/RobotContainer.java:39]: No register?

File: TeamCode/src/main/java/org/firstinspires/ftc/teamcode/Robot/commands/BulkReadingClear.java [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+    private final BulkReading bulkReading;
+
+    public BulkReadingClear(BulkReading bulkReading) {
+        this.bulkReading = bulkReading;
+    }
+
+    @Override
+    public void execute() {
+        for (LynxModule module : bulkReading.getAllHubs()) {
+            module.clearBulkCache();
```
Comment (foehammer127) [inline TeamCode/src/main/java/org/firstinspires/ftc/teamcode/Robot/commands/BulkReadingClear.java:18]: Not sure if this runs often enough

**Review by foehammer127 — CHANGES_REQUESTED** [review]
Based

File: TeamCode/src/main/java/org/firstinspires/ftc/teamcode/Robot/commands/BulkReadingClear.java (hunk 2 above)
Comment (cheeetoo) [reply TeamCode/src/main/java/org/firstinspires/ftc/teamcode/Robot/commands/BulkReadingClear.java:18]: [it should run every control cycle](https://github.com/FIRST-Tech-Challenge/FtcRobotController/blob/0879b4797f7c9664731c9306926f713ef4ddfe5d/FtcRobotController/src/main/java/org/firstinspires/ftc/robotcontroller/external/samples/ConceptMotorBulkRead.java#LL194C13-L194C13)

File: TeamCode/src/main/java/org/firstinspires/ftc/teamcode/Robot/RobotContainer.java (hunk 1 above)
Comment (cheeetoo) [reply TeamCode/src/main/java/org/firstinspires/ftc/teamcode/Robot/RobotContainer.java:40]: fixed

**Comment by foehammer127** [comment]
This is closed since it has behavior that we discussed in person.

**Pull request closed without merging**